bgcval2/analysis_compare.py

```diff
@@ -512,8 +537,12 @@ def load_comparison_yml(master_compare_yml_fn):
     # Image output settings:
     # dpi: pixels per inch (image resolution)
     # savepdf: also save the image as a pdf. 
+    # savecsv: Save the data that appears in the image.
     details['dpi'] = input_yml_dict.get('dpi', None)
     details['savepdf'] = input_yml_dict.get('savepdf', False)
+    details['savecsv'] = input_yml_dict.get('savecsv', False)
```

Comment:
Probably worth adding this functionality into documentation.

Reply:
yes!

Reply:
no IMHO since json is just the transport/payload, CSV is the final format no?

Reply:
human-readable and json are not sitting well on the same bus, bud :laughing:

Reply:
Too late! Changed it to `json`!

Reply:
whatever you think best, man - am merely devops here :grinning: